lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -70,6 +70,11 @@ defmodule LambdaEthereumConsensus.ForkChoice do
   def handle_cast({:on_block, block_root, %SignedBeaconBlock{} = signed_block, from}, store) do
     slot = signed_block.message.slot
 
+    Logger.info("[Fork choice] Adding new block ",
+      root: block_root,
+      slot: slot
+    )
```

Comment:
Nit
```suggestion
    Logger.info("[Fork choice] Adding new block", root: block_root, slot: slot)
```

Reply:
what is the change? formatting?

Reply:
Yes. And a trailing space in the log message.